cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -214,25 +214,22 @@ def _get_content_groups_usage_info(course, items):
         }
         """
         usage_info = {}
-        for item in items:
-            if hasattr(item, 'group_access') and item.group_access:
-                (__, group_ids), = item.group_access.items()
-                for group_id in group_ids:
-                    if group_id not in usage_info:
-                        usage_info[group_id] = []
-
-                    unit = item.get_parent()
-                    if not unit:
-                        log.warning("Unable to find parent for component %s", item.location)
-                        continue
-
-                    usage_info = GroupConfiguration._get_usage_info(
-                        course,
-                        unit=unit,
-                        item=item,
-                        usage_info=usage_info,
-                        group_id=group_id
-                    )
+        for item, group_id in GroupConfiguration._iterate_items_and_content_group_ids(course, items):
```

Comment:
Please update the docstring of this method `_get_content_groups_usage_info` about that it will return just the `cohort` groups (`CohortPartitionScheme`).